### ISSUE TITLE
Wrong user name in bullet point numbered 10 and under "Verify 'Deny log on as a service' GPO Settings"?

### ISSUE THREAD

**Issue by baardhermansen** [MicrosoftDocs/windowsserverdocs#4736]
On bullet point 10, which starts with "Configure the user rights to prevent the **BA** account...", shouldn't the correct text be "Configure the user rights to prevent the **administrator** account..."?

Same (potential) issue under the heading "Verify "Deny log on as a service" GPO Settings", under bullet point numbered 7;
"Click Browse, type the name of the BA account".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a861650d-0499-c46b-7ca8-d1fb04546192
* Version Independent ID: 3967b664-f419-6e8e-5ad8-f98cdcb21733
* Content: [Appendix D - Securing Built-In Administrator Accounts in Active Directory](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/plan/security-best-practices/appendix-d--securing-built-in-administrator-accounts-in-active-directory)
* Content Source: [WindowsServerDocs/identity/ad-ds/plan/security-best-practices/Appendix-D--Securing-Built-In-Administrator-Accounts-in-Active-Directory.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-ds/plan/security-best-practices/Appendix-D--Securing-Built-In-Administrator-Accounts-in-Active-Directory.md)
* Product: **windows-server**
* Technology: **ad-ds**
* GitHub Login: @MicrosoftGuyJFlo
* Microsoft Alias: **joflore**

**Comment by illfated** [comment]
Fair point, it looks a bit like the original author could have written the initials "BA" as a self reminder to treat the subject as a "Built-in Administrator" in contrast to Domain Administrator or any other administrator user type. Reading the text in the following images, it is quite clear that "BA" should be replaced by "Administrator" to match the end result of each numbered point.

(Please note that "bullet point" is usually a non-numbered entry prefaced by a big dot ("bullet"), so we usually talk about either "numbered lists" or "bullet point lists".)

**Comment by illfated** [comment]
I am also pondering whether to change the uppercase I to lowercase i in "Built-In Administrator" everywhere on the page, if the initials "BA" is a common initialism for "Built-in Administrator". Any feedback on that question is welcome too.

**Comment by baardhermansen** [comment]
> (Please note that "bullet point" is usually a non-numbered entry prefaced by a big dot ("bullet"), so we usually talk about either "numbered lists" or "bullet point lists".)

Good feedback, i'll take it to heart.

**Comment by baardhermansen** [comment]
> I am also pondering whether to change the uppercase I to lowercase i in "Built-In Administrator" everywhere on the page, if the initials "BA" is a common initialism for "Built-in Administrator". Any feedback on that question is welcome too.

I can only find the capital I in the page title and the first heading. It seems a bit strange having the letter capitalized there, so i agree about changing them.

**Comment by Xelu86** [comment]
#assign:xelu86

**Comment by Xelu86** [comment]
Thanks again for contacting Microsoft. We value your feedback. A documentation change has been made and will be live after the next business day. We will be closing this as resolved. Thank you and have a most pleasant day. #please-close
